backend/src/akvo/lumen/endpoint/collection.clj

```diff
@@ -1,6 +1,9 @@
 (ns akvo.lumen.endpoint.collection
   (:require [akvo.lumen.protocols :as p]
-            [akvo.lumen.lib.collection :as collection]            
+            [akvo.lumen.lib.collection :as collection]
+            [akvo.lumen.specs.components :refer (integrant-key)]
```

Comment:
Since we used vectors most of the places, is there a reason for using the list notation?